alifedata_phyloinformatics_convert/_aux/_alifestd_parse_ancestor_ids.py

```diff
@@ -0,0 +1,9 @@
+import typing
+
+
+def alifestd_parse_ancestor_ids(ancestor_list_str: str) -> typing.List[int]:
```

Comment:
turns out there was a parse ids implementation that was pasted all over. It's in _impl now so use and/or update that if posible.

Reply:
Updated in cdcd4dd